ftw/servicenavigation/resources/js/service-edit-form.js

```diff
@@ -1,44 +1,48 @@
-$(function() {
+require([
+  'jquery',
+], function($,){
```

Comment:
This will break on Plone 4.3 because you use `require` without `require.js`. Use the AMD design from here: https://github.com/umdjs/umd/blob/master/templates/amdWeb.js

```js
(function (root, factory) {
    if (typeof define === 'function' && define.amd) {
        // AMD. Register as an anonymous module.
        require('jquery', factory);
    } else {
        // Browser globals
        root.amdWeb = factory(root.jQuery);
    }
}(typeof self !== 'undefined' ? self : this, function ($) {
```

And ending the file with `}));` instead of `});` when using the approach above. 

PS: Trailing commas in the argument list of a function are only supported in ES2015 or later so that's why the Uglify.js couldn't uglify the compiled script.

Reply:
> 744d11a
👍